src/qtt/measurements/acquisition/uhfli_scope_reader.py

```diff
@@ -0,0 +1,303 @@
+
+    @sample_rate.setter
+    def sample_rate(self, value: float) -> None:
+        """ Sets the sample rate of the acquisition device.
+
+        Args:
+            value: The sample rate in samples per second.
+        """
+        self.__uhfli.scope_samplingrate_float.set(value)
+
+    @property
+    def period(self) -> float:
+        """ Gets the measuring period of the acquisition.
+
+        Returns:
+            The measuring period set value in seconds.
+        """
+        return self.__uhfli.scope_duration.get()
+
+    @period.setter
+    def period(self, value: float) -> None:
+        """ Sets the measuring period of the acquisition.
+
+        Args:
+            value: The measuring period in seconds.
+        """
+        self.__uhfli.scope_duration.set(value)
+
+    @property
+    def trigger_enabled(self) -> bool:
+        """ Gets the external triggering enabled status.
+
+        Returns:
+            The trigger enabled status set value.
+        """
+        is_enabled = self.__uhfli.scope_trig_enable.get()
+        if is_enabled == 'ON':
+            return True
+        if is_enabled == 'OFF':
+            return False
+        raise ValueError('Unknown trigger value ({})!'.format(is_enabled))
+
+    @trigger_enabled.setter
+    def trigger_enabled(self, value: bool) -> None:
+        """ Turns the external triggering on or off.
+
+        Args:
+            value: The trigger on/off value.
+        """
+        self.__uhfli.scope_trig_enable.set('ON' if value else 'OFF')
+
+    @property
+    def trigger_channel(self) -> str:
+        """ Gets the external triggering channel.
+
+        Returns:
+            The trigger channel value.
+        """
+        return self.__uhfli.scope_trig_signal.get()
+
+    @trigger_channel.setter
+    def trigger_channel(self, channel: str) -> None:
+        """ Sets the exernal triggering channel.
+
+        Args:
+            channel: The trigger channel value.
+        """
+        self.__uhfli.scope_trig_signal.set(channel)
+
+    @property
+    def trigger_level(self) -> float:
+        """ Gets the external triggering level in Volts.
+
+        Returns:
+            The trigger level in Volts.
+        """
+        return self.__uhfli.scope_trig_level.get()
+
+    @trigger_level.setter
+    def trigger_level(self, level: float) -> None:
+        """ Sets the exernal triggering level.
+
+        Args:
+            level: The external trigger level in Volts.
+        """
+        self.__uhfli.scope_trig_level.set(level)
+
+    @property
+    def trigger_slope(self) -> str:
+        """ Gets the external triggering slope.
+
+        Returns:
+            The scope trigger slope (e.g. Rise, Fall, etc.).
+        """
+        return self.__uhfli.scope_trig_slope.get()
+
+    @trigger_slope.setter
+    def trigger_slope(self, slope: str) -> None:
+        """ Sets the exernal triggering slope.
+
+        Args:
+            slope: The external trigger slope (e.g. Rise, Fall, etc.).
```

Comment:
etc is only two more options, `None` and `Both`, would be nice to have them included in the docstring